### ISSUE TITLE
Installation query

### ISSUE THREAD

**Issue by vrengad** [t0mer/green-api-custom-notifier#11]
I was trying to configure green API in my home assistant. I got struck at following step. Can you tell me which files I need to download. If possible can you provide some screenshots. 
 "Download the [green-api-custom-notifier](https://github.com/t0mer/green-api-custom-notifier), place it under the custom_components folder"

**Comment by vrengad** [comment]
I managed to figure it out. I created a folder named greenapi inside the custom_components directory, then downloaded the zip file, navigated to the greenapi folder inside, and uploaded the files into the folder I created. After checking the configuration and restarting Home Assistant, I added the configuration and restarted again. It worked fine after that. Earlier, I got an error when I tried updating the configuration without restarting, but everything looks good now.

**Comment by vrengad** [comment]
I managed to figure it out. I created a folder named greenapi inside the custom_components directory, then downloaded the zip file, navigated to the greenapi folder inside, and uploaded the files into the folder I created. After checking the configuration and restarting Home Assistant, I added the configuration and restarted again. It worked fine after that. Earlier, I got an error when I tried updating the configuration without restarting, but everything looks good now.
